app/components/feedback/container_component.html.erb

```diff
@@ -1,16 +1,18 @@
 <div class="p-2 border shadow-lg rounded-lg max-w-prose mx-auto flex-1">
-  <div class="flex justify-between items-center">
-    <%= heading(class: "text-lg") %>
-    <%= render Feedback::EditButtonComponent.new(feedback:, current_user: user) %>
-  </div>
-  <div>
-    <% if feedback.completed? %>
-      <div class="p-2">
-        <p><span class="font-bold">Rating:</span> <%= feedback.overall_rating %>/10</p>
-      </div>
-      <%= render Feedback::QuestionComponent.with_collection(questions) %>
-    <% else %>
-      <p class="p-4">Awaiting completion of feedback.</p>
-    <% end %>
-  </div>
+  <%= tag.turbo_frame(id: dom_id(feedback)) do %>
```

Comment:
Have to use this instead of the `turbo_frame_tag` helper because of some compatibility issues with ViewComponent. See [this documentation](https://viewcomponent.org/known_issues.html#turbo_frame_tag-double-rendering-or-scrambled-html-structure)